python/ray/air/utils/data_batch_conversion_utils.py

```diff
@@ -0,0 +1,90 @@
+from typing import Type
```

Comment:
air/util folder not utils

Reply:
Though this file in particular I think should be in air/_internal?

Reply:
These functions need to be used in Serve as well. I currently have them as DeveloperAPI

Regarding utils vs. util, let's do this in a sweeping change in a follow up PR.

Reply:
Just merged the PR doing the renaming.